lightning/src/ln/outbound_payment.rs

```diff
@@ -3121,6 +3142,46 @@ mod tests {
 			.unwrap()
 	}
 
+	fn blinded_path() -> BlindedMessagePath {
+		BlindedMessagePath::from_raw(
+			pubkey(40),
+			pubkey(41),
+			vec![
+				BlindedHop { blinded_node_id: pubkey(42), encrypted_payload: vec![0; 43] },
+				BlindedHop { blinded_node_id: pubkey(43), encrypted_payload: vec![0; 44] },
+			],
+		)
+	}
+
+	// FIXME(vincenzopalazzo): This function is duplicated with the one in static_invoive.rs
+	fn dummy_static_invoice() -> StaticInvoice {
```

Comment:
These could go in `offers/test_utils.rs`

Reply:
Thanks!